benchmarkList.md

```diff
@@ -207,8 +211,12 @@ DRB172-critical2-orig-no.c               |N2, N3 	 | Use of private and explicit
 DRB174-non-sibling-taskdep-no.c          |N3		 | Use of taskwait ensures no data race. 															| New
 DRB176-fib-taskdep-no.c                  |N3		 | Fibonacci code with proper task dependency.                                                                     						| Fibonacci code
 DRB182-atomic3_no.c                      |N3     | sync with busy wait loop using atomic                                               |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
-DRB184-bar1_no.c                         |N3     | 2-thread flag barrier using busy-wait loops and critical, no race                                            |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
-DRB186-bar2_no.c                         |N3     | uses locks to create a barrier that is used once                                             |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
-DRB188-bar3_no.c                         |N3     | implements 2-thread reuseable barrier using 3 locks, no race                                          |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
-DRB190-critsec2_no.c                     |N2, N3 | single producer single consumer with critical sections                                     |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
-DRB192-critsec3_no.c                     |N2, N3 | signal with busy wait loop using critical sections                                   |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
\ No newline at end of file
+DRB184-bar1_no.c                         |N3     | 2-thread flag barrier using busy-wait loops and critical, no race                   |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
```

Comment:
bar->barrier.  The filename is not that long. Use full words if possible to increase readability. @LChenGit